core/include/usChrono.h

```diff
@@ -20,12 +20,20 @@
 
 =============================================================================*/
 
-#include <usGlobalConfig.h>
 
-namespace us {
+#ifndef USCLOCK_H
```

Comment:
It is a new file with little contents and most a large part of it is the standard license header. So the standard Git similarity index value used to detect moves/renames kicks in and detects something. There's not much we can do here I think (you can change the similarity index when using git diff locally).